libraries/AP_HAL_ChibiOS/hwdef/common/stm32_util.c

```diff
@@ -468,3 +468,39 @@ uint32_t stack_free(void *stack_base)
     return ((uint32_t)p) - (uint32_t)stack_base;
 }
 #endif
+
+#ifdef RNG
```

Comment:
should have some way to enable/disable this feature, perhaps off by default?

Reply:
ChibiOS changes moved to separate PR https://github.com/ArduPilot/ardupilot/pull/17855